operator/config/samples/karmada.yaml

```diff
@@ -31,24 +31,36 @@ spec:
       serviceSubnet: 10.96.0.0/12
     karmadaAggregatedAPIServer:
       imageRepository: docker.io/karmada/karmada-aggregated-apiserver
-      imageTag: v1.7.0
+      imageTag: v1.8.0
       replicas: 1
     karmadaControllerManager:
       imageRepository: docker.io/karmada/karmada-controller-manager
-      imageTag: v1.7.0
+      imageTag: v1.8.0
       replicas: 1
     karmadaScheduler:
       imageRepository: docker.io/karmada/karmada-scheduler
-      imageTag: v1.7.0
+      imageTag: v1.8.0
       replicas: 1
     karmadaWebhook:
       imageRepository: docker.io/karmada/karmada-webhook
-      imageTag: v1.7.0
+      imageTag: v1.8.0
       replicas: 1
     kubeControllerManager:
       imageRepository: registry.k8s.io/kube-controller-manager
       imageTag: v1.25.4
       replicas: 1
+    karmadaMetricsAdapter:
+      imageRepository: docker.io/karmada/karmada-metrics-adapter
+      imageTag: v1.8.0
+      replicas: 2
+    # karmadaSearch: # used to install the `search` addons
+      # imageRepository: docker.io/karmada/karmada-search
+      # imageTag: v1.8.0
+      # replicas: 1
+    # karmadaDescheduler: # used to install the `descheduler` addons
+      # imageRepository: docker.io/karmada/karmada-descheduler
+      # imageTag: v1.8.0
+      # replicas: 1
```

Comment:
`karmada-operator` does not support `karmada-search` and `karmada-descheduler`, right?

Reply:
> support

The addon is not installed by default, if you want to install it, you need to show the definition in this file

Reply:
I'll add a note here to make this easier to understand